docs/source/components.rst

```diff
@@ -106,7 +106,7 @@ To create a beam splitter object with a given reflectivity value:
 >>> beam_splitter = BS(BS.r_to_theta(R))
 
 By default
-``theta`` is :math:`\pi/4`,
+``theta`` is :math:`\pi/2`,
```

Comment:
Should we consider that important enough to warn the user ?

Reply:
I think it was a doc mistake, so it's not mandatory.

Reply:
It 's a doc mistake. I fixed the default value above in the same page, and didn't check if the value further.. but the value was repeated elsewhere (here)!